webapp/channels/src/components/global_header/center_controls/user_guide_dropdown/user_guide_dropdown.tsx

```diff
@@ -139,7 +139,7 @@ class UserGuideDropdown extends React.PureComponent<Props, State> {
                     openLeft={false}
                     openUp={false}
                     id='AddChannelDropdown'
-                    ariaLabel={intl.formatMessage({id: 'sidebar_left.add_channel_dropdown.dropdownAriaLabel', defaultMessage: 'Add Channel Dropdown'})}
+                    ariaLabel={intl.formatMessage({id: 'channel_header.userHelpGuide', defaultMessage: 'Help'})}
```

Comment:
This used the label. I got rid of some redundant labels, so I changed it to use the same label that it uses for its button